### DISCUSSION TITLE
Offset calibration

### DISCUSSION THREAD

**Discussion by eben80** [dl9rdz/rdz_ttgo_sonde#195] [Developers corner]
I wanted to ask if this is still a feature? I can't see it in the config tab or file (showafc).
The reason I'm asking is because my ttgo with a March build has greatly superior reception over the current version and I wanted to check if it has anything to do with the calibration.

**Comment by dl9rdz** [comment]
If the screen file contains an entry for showing the measured error, it is now always shown. You can no longer disable that with the showafc setting.

The setting for the frequency correction value is still there.

**Reply by eben80** [reply]
Ah OK understood. So its still the case that whatever is displayed in that field, I should put in the offset section of config in Hz for the best reception. Right?

**Comment by dl9rdz** [comment]
Yes exactly. 

There are not really any changes in software since March that might influence sensitivity. At least no known issues.

**Reply by eben80** [reply]
OK thanks. It might be the GPS module that interferes somehow. The one with better reception doesn't have the GPS module.

**Reply by dl9rdz** [reply]
If its a T-Beam 1.0/1.1 you can test this hypothesis by disabling GPS power. Comment out axp.setPowerOutPut(AXP192_LDO3, AXP202_ON); in the code. Then GPS should be completely off.

A known source of potential interference is the TFT display. Shielding of the cables helps somewhat, or separating the antenna from the display.

Besides that, different kind of boards differ quite a lot. The old TTGO LoRa v1.0 are somewhat bad, and even worse are all of my T-Beam 0.7 in battery-only operation (no USB power; other people apparently had more luck with those).  Even within the same board type, some production quality variation has been observed.  I have seen two identical TTGO (even from the same order) in a LoRa-APRS application, and one of the two had greatly inferior reception.

**Reply by eben80** [reply]
Thanks for the tips. I only have the TTGO V2 1.6 boards. The GPS is a BN-180. 
![image](https://user-images.githubusercontent.com/2545460/135079980-827a8891-cc43-484c-ba6e-a476411962d6.png)
![image](https://user-images.githubusercontent.com/2545460/135079901-b959e477-37bd-4bc1-a8c7-3c809ab233c3.png)
The one with better reception is identical minus the buttons and GPS.
